Cargo.lock

```diff
@@ -3343,6 +3343,7 @@ version = "0.0.0"
 dependencies = [
  "object 0.34.0",
  "regex",
+ "similar",
```

Comment:
Could you check if you can just use the `diff` from the environment? As in, I think we already assume `diff` exists:

https://github.com/rust-lang/rust/blob/7bb4f0889e8b133c5b03c46f31f2ae6432c00219/tests/run-make/tools.mk#L21-L22

If that's not available, then I'm open to adding `similar`.

Reply:
> Would be nice to not have to rely on those external dependencies if possible/reasonable especially if they can have platform-specific differences and what not.

I think so too.